tests/test_app.py

```diff
@@ -21,4 +21,4 @@
 def test_discovery(client: RequestHandler) -> None:
     response = client.get("/.well-known/terraform.json", headers={"host": "localhost"})
     assert response.status_code == HTTPStatus.OK
-    assert response.json == {"modules.v1": "https://localhost/v1"}
+    assert response.json == {"modules.v1": "/"}
```

Comment:
Please try to mimic the results from `https//registry.terraform.io/.well-known/terraform.json`

```json
{
  "modules.v1": "/v1/modules/",
  "providers.v1": "/v1/providers/"
}
```

Reply:
Keeping in mind that "v1" *is not* to be hard-coded in the codebase and it is related to the "stage" where the api was mounted.